### DISCUSSION TITLE
Welcome to scPagwas Discussions!

### DISCUSSION THREAD

**Discussion by dengchunyu** [sulab-wmu/scPagwas#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by czddaniel** [comment]
您好，请问一下可以问您要一下邮件地址嘛，我想发给您一些问题

**Comment by MAKABAKA2000-11-22** [comment]
Hello , I would like to ask if there are any parameter modification species for single-cell mice data
